Sources/Filters/Sources/PlaneSource/index.js

```diff
@@ -129,23 +129,22 @@ function vtkPlaneSource(publicAPI, model) {
           theta = 180.0;
           vtkMath.subtract(model.point1, model.origin, rotationVector);
         } else {
-          vtkMath.cross(model.normal, normal, rotationVector);
+          vtkMath.cross(model.normal, n, rotationVector);
           theta = vtkMath.degreesFromRadians(Math.acos(dp));
         }
         // Create rotation matrix
         const transform = mat4.create();
         const negCenter = [];
-
         vec3.negate(negCenter, model.center);
 
-        mat4.translate(transform, transform, negCenter);
```

Comment:
`vec3.negate` doesn't modify model.center. Only `negCenter`. As `negCenter` is not used, you can remove `vec3.negate(negCenter, model.center)`

Reply:
My bad, didn't see it's used right after.